### PR TITLE
Update docusaurus

### PR DESCRIPTION
**Pull request by hobadams**
Update to latest version of Docusaurus

This includes:

- PWA support
- Broken link checker 

The broken link tool has also highlighted a tonne of links that needed fixing :-)

### CODE REVIEW

File: docs/open-source/falcon-client/code-splitting.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default () => (
 )
 ```
 
-When you create your project using [`create-falcon-app`](getting-started/installation#create-falcon-app-cli-tool) Falcon provides a wrapper function for `loadable`. This wrapper function shows a spinner while the component is loading to indicate to the user that the contents of the page are currently being fetched. Using this wrapper function is optional, but it is recommended to provide a fallback state to `loadable` if you choose to write your own.
+When you create your project using [`create-falcon-app`](/docs/open-source/getting-started/installation#create-falcon-app-cli-tool) Falcon provides a wrapper function for `loadable`. This wrapper function shows a spinner while the component is loading to indicate to the user that the contents of the page are currently being fetched. Using this wrapper function is optional, but it is recommended to provide a fallback state to `loadable` if you choose to write your own.
```
Comment (slorber) [inline docs/open-source/falcon-client/code-splitting.md:31]: totally unrelated to your pwa problem but did doc linking break?

Comment (hobadams) [reply docs/open-source/falcon-client/code-splitting.md:31]: > totally unrelated to your pwa problem but did doc linking break?

It didn't break but relative links were flagged by the broken link checker.

Comment (slorber) [reply docs/open-source/falcon-client/code-splitting.md:31]: I see, will try to not report this.

I think if you reference the md file (with extension) it wouldn't be flagged

Comment (hobadams) [reply docs/open-source/falcon-client/code-splitting.md:31]: That's good to know. Thanks

Comment (slorber) [reply docs/open-source/falcon-client/code-splitting.md:31]: FYI this broken link will be fixed in next version so you can revert to relative links if you want

Comment (hobadams) [reply docs/open-source/falcon-client/code-splitting.md:31]: Thanks Slorber. That's great to know

Comment (slorber) [reply docs/open-source/falcon-client/code-splitting.md:31]: that should be fixed in alpha 60. Would be interested to know if you can safely revert those changes

File: docusaurus.config.js [hunk 2]
```diff
@@ -1,180 +1,200 @@
...
 
 module.exports = {
-  title: 'DEITY Falcon',
-  tagline: 'Documentation',
-  url: 'https://deity.io',
-  baseUrl: '/',
-  favicon: 'img/favicon.png',
-  organizationName: 'DEITY', // Usually your GitHub org/user name.
-  projectName: 'falcon-platform-docs', // Usually your repo name.
+  title: "DEITY Falcon",
```
Comment (nimobeeren) [inline docusaurus.config.js:4]: Please revert the quote style changes in this file to be consistent

**Comment by slorber** [comment]
@nimobeeren alpha 60 is already released

**Comment by nimobeeren** [comment]
@slorber oh awesome! Thanks so much for helping out by the way!

**Comment by hobadams** [comment]
@slorber - I've just tested the relative links and they work well. Thank you for all your help!

**Comment by hobadams** [comment]
@EddyVinck @Borales - this is ready to look at when ever you have time

**Pull request merged**